docs/kb/semgrep-supply-chain/why-no-findings.md

```diff
@@ -0,0 +1,78 @@
+---
+description: Troubleshoot why findings for Semgrep Supply Chain are not showing.
+tags:
+  - Semgrep Supply Chain
+---
+# Why aren't Supply Chain findings showing?
+
+## Ensure compatibility
+
+First, verify that your repository meets the basic requirements for Semgrep Supply Chain:
+
+### Are the language and ecosystem supported?
+
+Check the [Supported Languages table](/docs/supported-languages/#general-availability) for support for the project's language and ecosystem, as well as any ecosystem-specific requirements.
+
+###  Is the supported lockfile present in an appropriate location with the expected name?
+
+Semgrep Supply Chain searches the parent directories of any files for a language for the nearest relevant lockfile. This typically means that monolithic repositories (monorepos) are recognized by the presence of the lockfiles in subdirectories.
+
+Semgrep Supply Chain only recognizes the lockfile names indicated in the [Supported Languages table](/docs/supported-languages/#general-availability).
+
+### Does the lockfile contain dependencies with exact versions?
+
+If your dependency file is a [manifest file](/docs/semgrep-supply-chain/glossary/#manifest-file) and does not specify exact (pinned) versions for all dependencies, Semgrep Supply Chain does not report vulnerabilities for the dependencies that are not pinned. An unpinned dependency may already be installed at a safe version for a particular [Advisory](https://semgrep.dev/docs/semgrep-supply-chain/glossary/#advisory).
+
+Pinned dependencies can be analyzed even if the file contains other unpinned dependencies. Manifest files can also be helpful to determine whether a dependency is [transitive](/docs/semgrep-supply-chain/glossary/#transitive-or-indirect-dependency).
+
+## Check scan status and result location
+
+Next, ensure that the scan was successful and sent results to the expected location.
+
+### Did the scan run successfully?
+
+If the scan did not complete, or failed when trying to send results, the dependency information will not be available.
+
+Review the logs from the scan and determine whether it was successful, or ran into an issue. Common issues include:
+
+* The lockfile could not be parsed successfully. The CI output should point to the line where the error occurred.
```

Comment:
That's a nice idea! Do you have a good example I can add? I haven't run into this so far.

In regards to the CTA, there is one at the bottom of the article, will that work? I could also put a link to move down to that section here, since if you get a parsing error the rest is probably moot.

Reply:
<img width="882" alt="Screen Shot 2023-06-29 at 12 59 11 PM" src="https://github.com/returntocorp/semgrep-docs/assets/100232702/49bce721-3211-44a2-af12-8df382551411">

something like this maybe?

Reply:
For CTA, I like the latter! Yeah, let's just get folks immediately to the CTA + reach out for help phase if they're truly stuck

Reply:
Thanks! I'll get those updates done now.